devicetypes/smartthings/zigbee-dual-switch.src/zigbee-dual-switch.groovy

```diff
@@ -0,0 +1,140 @@
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Health Check"
+        capability "Switch"
+        
+        command "childOn", ["string"]
+        command "childOff", ["string"]
+        
+        fingerprint  profileId: "0104", inClusters: "0000, 0005, 0004, 0006", manufacturer: "0000000", model: "cc891c853291418daa4e26cb315a3a0c", deviceJoinName: "Orvibo Smart Switch 1"
+       }
+    // simulator metadata
+    simulator {
+        // status messages
+        status "on": "on/off: 1"
+        status "off": "on/off: 0"
+
+        // reply messages
+        reply "zcl on-off on": "on/off: 1"
+        reply "zcl on-off off": "on/off: 0"
+    }
+
+    tiles(scale: 2) {
+        multiAttributeTile(name:"switch", type: "lighting", width: 6, height: 4, canChangeIcon: true){
+            tileAttribute ("device.switch", key: "PRIMARY_CONTROL") {
+                attributeState "on", label:'${name}', action:"switch.off", icon:"st.switches.light.on", backgroundColor:"#00A0DC", nextState:"turningOff"
+                attributeState "off", label:'${name}', action:"switch.on", icon:"st.switches.light.off", backgroundColor:"#ffffff", nextState:"turningOn"
+                attributeState "turningOn", label:'${name}', action:"switch.off", icon:"st.switches.light.on", backgroundColor:"#00A0DC", nextState:"turningOff"
+                attributeState "turningOff", label:'${name}', action:"switch.on", icon:"st.switches.light.off", backgroundColor:"#ffffff", nextState:"turningOn"
+            }
+        }
+        standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
+        }
+        main "switch"
+        details(["switch", "refresh"])
+    }
+}
+
+def installed() {
+         createChildDevices()
+         configure()
+}
+
+// Parse incoming device messages to generate events
+def parse(String description) {
+    log.debug "description is $description"
+    Map map = zigbee.getEvent(description)
+    if (map) {
+        if (description?.startsWith('on/off')) {
+            log.debug "receive on/off message without endpoint id"
+        } else {
+            Map descMap = zigbee.parseDescriptionAsMap(description)
+            log.debug "$descMap"
+
+            if (descMap?.clusterId == "0006" && descMap.sourceEndpoint == "01") {
+                def result = map ? createEvent(map) : [:]
+                sendEvent(result)
+             } else if (descMap?.clusterId == "0006" && descMap.sourceEndpoint == "02") {
+                def result = map ? createEvent(map) : [:]
+                def childDevice = childDevices.find {
+			it.deviceNetworkId == "$device.deviceNetworkId-ep2"
```

Comment:
After talking internally I think we would prefer the format of child network IDs to be "<parent DNI>:<%02X ep id>" So for a device with short ID of "CAFE" it would be "CAFE:01" and "CAFE:02" for the children